common/remote/hooks/git/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@
 
 project( gitfile )
 
-include (libgit2.cmake)
+find_package(unofficial-git2 CONFIG REQUIRED)
```

Comment:
Should I be concerned by the "unofficial" here?

Reply:
Nope.